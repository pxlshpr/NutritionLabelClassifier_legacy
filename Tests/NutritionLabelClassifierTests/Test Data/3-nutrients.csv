attributeString,value1String,value2String
energy,94kcal,75kcal
protein,5.4g,4.3g
fat,0.1g,0.1g
saturatedFat,0.1g,0.1g
transFat,0g,0g
cholesterol,4mg,3mg
carbohydrate,17.8g,14.2g
dietaryFibre,0.1g,0.1g
sodium,84mg,67mg
calcium,225mg,180mg