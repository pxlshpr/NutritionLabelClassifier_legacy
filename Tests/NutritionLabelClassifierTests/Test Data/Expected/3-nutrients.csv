attributeString,value1String,value2String
energy,94kcal,75kcal
protein,5.4g,43g
fat,0.1g,0.1g
saturatedFat,0.1g,0.1g
transFat,0g,0g
cholesterol,4mg,3mg
carbohydrate,17.8g,14.2g
dietaryFibre,0.1g,0.19g
sodium,84mg,67mg
calcium,225mg,180mg
servingsPerContainer,8,
